stl/inc/iterator

```diff
@@ -155,8 +176,8 @@ public:
     }
 
 protected:
-    _Container* container; // pointer to container
-    typename _Container::iterator iter; // iterator into container
+    _Container* container              = nullptr;

```

Comment:
This is failing clang-format; consider enabling format-on-save in your editor. Everything else looks good.